packages/patternfly-4/react-core/src/components/Pagination/Navigation.tsx

```diff
@@ -54,6 +56,7 @@ export class Navigation extends React.Component<NavigationProps, NavigationState
   static defaultProps = {
     className: '',
     lastPage: 0,
+    firstIndex: 0,
```

Comment:
Don't think we need this here. Did you mean `firstPage`?

Reply:
Yup, at first I passed `firstIndex` and renamed it to `firstPage` good catch!